src/strided.jl

```diff
@@ -0,0 +1,172 @@
+#-------------------------------------------------------------------------------------------
+# Force strided implementation on AbstractArray instances with HPTTBLAS backend
+#-------------------------------------------------------------------------------------------
```

Comment:
No, results from copying my HPTTBLAS implementation, which is also ready, but doesn't work on all test cases because it needs one of the strides of the input array to be 1.